src/Server/Resource/DoctrineResource.php

```diff
@@ -2,25 +2,28 @@
 
 namespace ZF\Apigility\Doctrine\Server\Resource;
 
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\EntityManagerInterface;
```

Comment:
This file is designed to handle ORM and ODM.  You'll need to not code for ORM only.